rllib/core/rl_module/tf/tf_rl_module.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Mapping
+
+from ray.rllib.core.rl_module import RLModule
+from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+
+
+_, tf, _ = try_import_tf()
+
+
+class TFRLModule(RLModule, tf.keras.Model):
+    def __init__(self, config: Mapping[str, Any]) -> None:
```

Comment:
docstring

Reply:
in the new world, the base `TfRLModule` won't need any arg passed to its `__ini__.py`